spring-addons-starter-oidc/src/main/java/com/c4_soft/springaddons/security/oidc/starter/reactive/resourceserver/ReactiveSpringAddonsOidcResourceServerBeans.java

```diff
@@ -1,6 +1,7 @@
 package com.c4_soft.springaddons.security.oidc.starter.reactive.resourceserver;
 
 import java.nio.charset.Charset;
+import java.sql.Date;
```

Comment:
I think we want `java.util.Date` here.

Reply:
Gotcha - good catch. IntelliJ auto import and I should have caught that while committing.